services/base/repository.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   interval: 5m
   ref:
-    branch: main
+    branch: aa/aso-initial
```

Comment:
@aadev1 : Does branch name need to change?

Reply:
reset to main